examples/gguf-hash/deps/xxhash/xxhash.h

```diff
@@ -5495,7 +5497,9 @@ XXH3_accumulate_512_sve( void* XXH_RESTRICT acc,
     const uint64_t *xinput = (const uint64_t *)(const void *)input;
     const uint64_t *xsecret = (const uint64_t *)(const void *)secret;
     svuint64_t kSwap = sveor_n_u64_z(svptrue_b64(), svindex_u64(0, 1), 1);
-    uint64_t element_count = svcntd();
+    if (!ggml_sve_cnt_b)
+        ggml_sve_cnt_b = PR_SVE_VL_LEN_MASK & prctl(PR_SVE_GET_VL);
+    uint64_t element_count = ggml_sve_cnt_b / 8;
```

Comment:
I don't think that we should be modifying this file, since it is a 3rd party library. It would be better to submit the fix upstream.